README.md

```diff
@@ -1 +1,41 @@
-# To-Do-List
\ No newline at end of file
+# To-Do-List
+
+> The project focused on creating To do list structure
+
+## Built With
+- HTML
+- CSS
+-JAVASCRIPT
+​
+## Live Demo (if available)
+
+- https://alphantihinduka.github.io/Alpha-Mobile-portifolio/
+
+## Getting Started
+
+-Install webpack
+
+-Used the microverse linter documentation
+
+
+-set up linter for HTML and CSS
+
+-Install eslintrc.json.
+
+## Authors
+
+
+👤 **Author1**
+
```

Comment:
- [ ] (Optional) It would be nice to include your name here instead of **Author1**. That would make the `README.md` more personalized and professional. 👍